src/Engine/Graphics/Outdoor.cpp

```diff
@@ -2116,6 +2116,20 @@ void ODM_ProcessPartyActions() {
         }
     }
 
+    if (pParty->uFlags & PARTY_FLAGS_1_LANDING) {
```

Comment:
Please add a comment before this code block so that we don't pull our hair two years down the line trying to figure out what's going on